modules/031-linstor/images/drbd-driver-loader/Dockerfile

```diff
@@ -2,30 +2,36 @@ ARG BASE_DEBIAN_BULLSEYE
 
 FROM $BASE_DEBIAN_BULLSEYE as builder
 ARG DRBD_GITREPO=https://github.com/LINBIT/drbd
-ARG DRBD_VERSION=9.1.7
+ARG DRBD_VERSION=9.1.8
 
 RUN apt-get update \
  && apt-get install -y make git \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
+# Using source code from GitHub repository
 RUN git clone ${DRBD_GITREPO} /drbd \
  && cd /drbd \
  && git reset --hard drbd-${DRBD_VERSION} \
  && make tarball \
- && mv ./drbd-*.tar.gz /drbd.tar.gz \
- && mv ./docker/entry.sh /entry.sh \
+ && mv ./drbd-*.tar.gz /drbd.tar.gz
+
+# # Using source code provided by LINBIT
+# RUN DRBD_PKG=https://pkg.linbit.com//downloads/drbd/9/drbd-${DRBD_VERSION}.tar.gz \
+#  && wget -O /drbd.tar.gz ${DRBD_PKG}
+
```

Comment:
The comments seems to be useless

Reply:
There are two options where you can get drbd source code package:

1. Build from Github repository
2. Take from LINBIT's archive

Final packages are slightly different, since LINBIT tarballs also induce compatibility patches. I would like to not removing this comment to have an ability simple swapping between them.